src/components/Header.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = (props) => {
+  return (
+    <header>
+      <div>
+        <Link to='/'>
+          Adopt A Family
+        </Link>
+      </div>
+      <div className="headerLinks">
+        <div className="headerMission">
```

Comment:
Update class names to follow lowercase-hyphenated naming conventions.

ie: header-links, header-mission, etc.